lib/webclient.js

```diff
@@ -36,13 +36,16 @@ class WebClient {
 				if (response_status === 429) {
 					console.log(`CODELIGHTHOUSE: you have reached your rate limit of errors for the month. Please upgrade your plan for more.`)
 				}
-				else if (response_status === 200) {
+				else if (response_status === 200 || response_status === 201) {
 					console.log(`CODELIGHTHOUSE: error_guid=${response_data['error_guid']}`)
 				}
-				else if (response_status !== 200 || self.debug) {
+				else if (response_status !== 200 && response_status !== 201 && response_status !== 429) {
 					console.log(`CODELIGHTHOUSE: returned message ${response_data}`);
```

Comment:
Do you want to do the response codes changes here?

Reply:
What? It already checks.

Reply:
can you clarify what you mean @willtraweek ?